### ISSUE TITLE
How to restart nanohttpd server after an exception in side.

### ISSUE THREAD

**Issue by sakkeerhussain** [NanoHttpd/nanohttpd#472]
Issue in one request fails my server and unable to serve continuously. Is there an option to restart server after a crash inside?

**Comment by LordFokas** [comment]
This needs to be looked at. With proper exception handling your server never really stops though.

**Comment by sakkeerhussain** [comment]
@LordFokas I am able to handle all exception inside with a try catch. But in case of client disconnected and time out exceptions (Socket write exception), how can we(users of nanohttpd server) catch exceptions?

**Comment by LordFokas** [comment]
Oh that was something that was kind of half assed in v2 but I think it has been fixed, at least as of the code in v3. Will add to the milestone just foor the sake of double checking :)

**Comment by sakkeerhussain** [comment]
@LordFokas I am using version 2.3.1 and having that issue. Any update available for that version?

**Comment by LordFokas** [comment]
not really, no. V3 is badly stuck as of now, and I'm too busy trying to get my personal life rolling to get it going for the time being :/
